.github/workflows/manifests.yml

```diff
@@ -2,6 +2,9 @@
 name: manifests
 
 on:
+  push:
+    branches:
+      - 'manifests-jdks'
```

Comment:
Some background regarding this branch would help.

Reply:
I forget. removed.